wigwam/cli/build_commands.py

```diff
@@ -170,6 +173,52 @@ def init_build_parsers(subparsers: argparse._SubParsersAction) -> None:
         ' Defaults to "build-installed".',
     )
 
+    parser_build_all = subparsers.add_parser(
+        "build-all",
+        parents=[config_params, no_cache_params],
+        help="Performs the complete compilation process, from initial GitHub checkout "
+        "to installation.",
+        formatter_class=help_formatter,
+    )
+    parser_build_all.add_argument(
+        "--base",
+        "-b",
+        type=str,
+        default="setup-mamba-dev",
+        help='The name of the parent Docker image. Default is "setup-mamba-dev".',
+    )
+    parser_build_all.add_argument(
+        "--tag",
+        "-t",
+        default="build",
+        type=str,
+        help="The sub-prefix of the Docker images to be created. Generated images will "
+        f'have tags fitting "{universal_tag_prefix()}-[TAG]-*". Default: "build"',
+    )
+    parser_build_all.add_argument(
+        "--copy-path",
```

Comment:
Let's rename this parameter to `--src-path` to be a little more clear that it should be the source prefix path

```suggestion
        "--src-path",
```